src/test/java/org/zalando/nakadi/service/subscription/StreamingContextTest.java

```diff
@@ -130,27 +127,4 @@ public void onExit() {
         Assert.assertArrayEquals(new boolean[]{true, true}, onExitCalls);
     }
 
-    @Test
-    public void testOnNodeShutdown() throws Exception {
```

Comment:
This is not good, since this confirms the sequence of the actions for subscription clean up in case of instance termination.